src/protos/io/defang/v1/fabric.proto

```diff
@@ -326,6 +331,22 @@ message ListConfigsRequest {
 
 message ListConfigsResponse { repeated ConfigKey configs = 1; }
 
+message ListDeploymentsRequest {
+  string project = 1;
+}
+
+message Deployment {
```

Comment:
```suggestion
message DeploymentRequest {
```

Reply:
Check gRPC naming guidelines

Reply:
Should I have a `Deployment` message in addition to a `DeploymentRequest`? For example, in this case, which should I use?

```diff
 message ListDeploymentsResponse {
-  repeated Deployment deployments = 1;
+  repeated DeploymentRequest deployments = 1;
 }
```

Reply:
> Check gRPC naming guidelines

Do you have a specific resource in mind? If I search GRPC naming guidelines, I find [this document](https://github.com/grpc/grpc/blob/master/doc/naming.md), which I don't think directly relates to your comment.